hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/OmSnapshotUtils.java

```diff
@@ -65,6 +66,12 @@ public static Object getINode(Path file) throws IOException {
     return Files.readAttributes(file, BasicFileAttributes.class).fileKey();
   }
 
+  public static String getInodeAndMtime(Path file) throws IOException {
```

Comment:
```suggestion
/**
 * Returns a string combining the inode (fileKey) and the last modification time (mtime) of the given file.
 * <p>
 * The returned string is formatted as "{inode}-{mtime}", where:
 * <ul>
 *   <li>{@code inode} is the unique file key obtained from the file system, typically representing the inode on POSIX systems</li>
 *   <li>{@code mtime} is the last modified time of the file in milliseconds since the epoch</li>
 * </ul>
 *
 * @param file the {@link Path} to the file whose inode and modification time are to be retrieved
 * @return a string in the format "{inode}-{mtime}"
 * @throws IOException if an I/O error occurs
 */
  public static String getInodeAndMtime(Path file) throws IOException {
```

Reply:
sugest a more descriptive method name. For instance, `getFileInodeAndLastModifiedTimeString()`

Reply:
done.